feature/experimental/p4rt/README.md

```diff
@@ -10,12 +10,26 @@ This document specifies the requirements for p4rt test implementation.
     1.  WBB P4 Protobuf file:
         https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/p4rt/wbb.p4info.pb.txt
 
-3.  The client should make use of Ondatra Raw API
-    `dut.RawAPIs().P4RT().Default(t)`
+3.  Tests should instantiate new P4RT clients using
+    `p4rt_client.NewP4RTClient()` function. This instantiates the
+    `StreamTermErr` channel required to investigate errors when the p4rt stream
+    terminates.
```

Comment:
```suggestion
3.  Tests should create new P4RT clients using the
    `p4rt_client.NewP4RTClient()` function, which sets up the
    `StreamTermErr` channel required to check errors when the p4rt stream
    terminates.
```

Just a nit to use shorter words because they are easier to read.